README.md

```diff
@@ -52,6 +52,47 @@ npm install @google-cloud/recaptcha-enterprise
 ```
 
 
+### Using the client library
+
+```javascript
+/**
+ * Instantiate the reCAPTCHA Enterprise client and make request.
+ * @param {string} projectNumber The project number to use when making requests.
+ */
+async function quickstart(projectNumber) {
```

Comment:
So `projectNumber` and `projectId` are very different things.  I am mostly sure you want `projectId` here.